tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -62,20 +62,42 @@ func setUpTestDir() error {
 	return nil
 }
 
-func mountGcsfuse() error {
+func mountCommand(flag string) string {
+	cmd := binFile + " --debug_gcs" + " --debug_fuse " + flag + " " + *testBucket + " " + mntDir + "\n"
+	return cmd
+}
+
+func mountGcsfuse(flag string) error {
+	// Adding mount command in logFile
+	file, err := os.OpenFile(logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+
+	if err != nil {
+		fmt.Println("Could not open logfile")
+	}
+
+	defer file.Close()
+
+	_, err2 := file.WriteString(mountCommand(flag))
+
+	if err2 != nil {
+		fmt.Println("Could not write cmd to logFile")
+	}
+
 	mountCmd := exec.Command(
```

Comment:
please print using this mountCmd. no need of seperate method to create command string for logging

Reply:
fixed

Reply:
fixed